src/app/components/left-menu/left-menu.component.ts

```diff
@@ -80,47 +80,47 @@ export class LeftMenuComponent {
 
   public pages: Page[] = [
     {
-      name: 'Map',
+      name: this.translate.instant('Map'),
```

Comment:
I think we should the pipe translate instead
One of the source problems of translation not getting updated was the use of `translate.instant` instead of the pipe

Reply:
Fixed